addons/web_tour/static/src/js/tour_disable.js

```diff
@@ -0,0 +1,26 @@
+odoo.define('web_tour.DisableTour', function (require) {
+"use strict";
+
+var TourManager = require('web_tour.TourManager');
+
+TourManager.include({
+    /**
+     * Disable tours if Odoo installed with demo data.
+     *
+     * @override
+     * @private
```

Comment:
According to the guidelines ``@override`` allows to avoid repeating the params and returns declarations.